application/templates/pages/about/performance.md

```diff
@@ -0,0 +1,39 @@
+Performance indicators show how the planning data platform is making it easier to find, use and trust planning and housing data in England.
+
+Last updated 22 July 2024.
+
+## Availability
+
+We index planning and housing datasets provided by multiple organisations in England. Many datasets have one owner providing the data nationally. 8 datasets are compiled from local planning authorities across England. We’re working with 73 out of 311 local planning authorities to start providing that data nationally.
+
+### 89 datasets on the platform
+
+?? national datasets available from ?? organisations. ? added last month (+/-0%).
+
```

Comment:
Current platform total:
 - 130 out of 584 data sources for 8 datasets provided by 31 out of 73 local planning authorities
Added last week
 - ? new data sources. +/-0% change from previous week
 - ? new authorities providing data. +/-0% change from previous week
Dataset and provider average completeness
For 8 datasets:
 - Average of ?? providers out of expected 311 (??%). +/-0% change from previous week
 - Average of ? out of 8 datasets per provider. +/-0% change from previous week